types/react-request/index.d.ts

```diff
@@ -1,51 +1,73 @@
 // Type definitions for react-request 3.1
 // Project: https://github.com/jamesplease/react-request
 // Definitions by: Danny Cochran <https://github.com/dannycochran>
+//                 Angus Fretwell <https://github.com/angusfretwell>
```

Comment:
Is there a reason you are adding this individual to the `Definitions by:` list?

Reply:
@rbuckton, we collaborated these definitions for our own project :)